League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -117,68 +138,171 @@ private void startCloningRepositories()
                     }
                 };
                 options.BranchName = "Indev";
-                options.RecurseSubmodules = true;
+                options.RecurseSubmodules = false;                
```

Comment:
Why? Is there a problem with this recursive cloning?

Reply:
Yeah, everytime I ran it, it crashed.